### ISSUE TITLE
Fossil Collection Event - attributes to display for Details tab, map pop-up and search card

### ISSUE THREAD

**Issue by LeannePyle** [bcgov/BCHeritage#591]
Fossil Collection Event Details tab

- [x] Header = (Year) - Location Descriptor or 1:50K mapsheet name
- [x] Detailed Location
- [ ] --thumbnail map of the location at the top
- [ ] ~Detailed Location~ -- duplicate
- [x] Spatial Coordinates
- [x] 1:50K mapsheet name
- [x] 1:50K mapsheet number
- [x] 1:250K mapsheet name
- [x] Collection Year 
- [x] Collector(s)
- [x] Samples Collected – Repository or field ID, Scientific name or common name
- [x] Fossil Abundance
- [x] Significance Ranking Code

Fossil Collection Event map pop-up

- [x] Header = (Collection Year) – Location Descriptor or 1:50K mapsheet name
- [x] Detailed Location
- [x] Formation (if applicable)
- [x] Period

Fossil Collection Event Search Result card

- [x] Header = (Collection Year) – Location Descriptor or 1:50K mapsheet name
- [x] Detailed Location
- [x] Period
- [x] Samples Collected – Scientific name or common name

**Comment by bferguso** [comment]
@LeannePyle, @elisabethdeom  
- Are you looking for the Formation(s) as documented in the collection event samples?
- by "Period" do you mean Stratigraphy? Can you please clarify what exactly you'd like displayed?

**Comment by LeannePyle** [comment]
Yes, the Formation(s) from the Stratigraphy of the Fossil Sample Resource Model
Period from the Geological Age of the Fossil Sample Resource Model

**Comment by bferguso** [comment]
<img width="604" alt="image" src="https://github.com/bcgov/BCHeritage/assets/6750642/ca0c2faa-8427-4181-8abe-95435d28f889">

**Comment by bferguso** [comment]
This is done, but has brought up an issue that if a resource this Collection Event is dependant on (in this case Fossil Sample) is updated, it does't update the descriptors of the dependant object. Opening a ticket for that (#858)

**Comment by bferguso** [comment]
Descriptor refresh resolved.

**Comment by bferguso** [comment]
For the samples collected, for all samples as a whole, listing scientific names if any found, otherwise listing common names. Is this OK? Or shoudl this be done on a sample-by-sample basis? (Eg 3 samples, one has scientific names, 2 only have common names, only the scientific names from the first sample will be listed).

**Comment by LeannePyle** [comment]
Yes to list scientific names when known. And in the event of multiple samples, default to listing the scientific names

**Comment by bferguso** [comment]
Automated cleanup
